server/models/postgis/task.py

```diff
@@ -72,7 +72,7 @@ def delete(self):
         db.session.commit()
 
     @staticmethod
-    def update_task_locked_with_duration(project_id: int, task_id: int, lock_action):
+    def update_task_locked_with_duration(task_id: int, project_id: int, lock_action: TaskStatus, user_id: int):
```

Comment:
I know it would be good to match the order of the calls, but it was easier for now to switch back to the original style given changes in #1223